src/components/NewCommentForm.tsx

```diff
@@ -14,24 +87,30 @@
             name="name"
             id="comment-author-name"
             placeholder="Name Surname"
-            className="input is-danger"
+            className={classNames('input', { 'is-danger': isErrorName })}
+            value={name}
+            onChange={handleNameChange}
```

Comment:
It's a good practice to follow naming conventions for methods. So it can be `handleNameInputChange` instead of `handleNameChange`.